docs/tutorials/basics/part1.md

```diff
@@ -100,42 +100,104 @@ of **changes** to the view, but cannot inspect its current contents.
 
 Click the PLAY button to run the pipeline.
 
-## Step 2. Populate tables manually
+## Step 2. Insert data
 
-:::info UNDER CONSTRUCTION
+When the pipeline is running it can process incoming changes. Let's try just that via ad-hoc queries.
 
-This section is under construction.
+Open Ad-hoc query tab and insert in the input field the following statement:
 
-:::
+```sql
+INSERT INTO VENDOR (id, name, address) VALUES
+(1, 'Gravitech Dynamics', '222 Graviton Lane'),
+(2, 'HyperDrive Innovations', '456 Warp Way'),
+(3, 'DarkMatter Devices', '333 Singularity Street');
+```
+
+Press `Enter` or click the Play button to submit the query. You will see the result containing the number of inserted rows: 3.
+
+![Query to insert vendor data](basics-part1-1.png)
+
+Now let's check the state of the table with a quick select query that we insert into the empty input field below the result from the first query:
+
+```sql
+SELECT * FROM VENDOR
+```
+
+![Inserted vendor data](basics-part1-2.png)
+
+Yep, everything is in order. Let's fill the other table:
+
+```sql
+INSERT INTO PART (id, name) VALUES
+(1, 'Flux Capacitor'),
+(2, 'Warp Core'),
+(3, 'Kyber Crystal')
```

Comment:
Please add semicolons to all queries. We don't require them now, but we may in the future, and it's the standard behavior for SQL CLIs.

Reply:
Left them out on purpose, but will change to include them if that's the case